### DISCUSSION TITLE
Latest working commit hash of AUTOMATIC1111's UI

### DISCUSSION THREAD

**Discussion by LiamPerson** [d8ahazard/sd_dreambooth_extension#836] [Show and tell]
Share the latest working commit hash of AUTOMATIC1111's UI here so people who just want to use the plugin can plug-and-play 😇👇

**Comment by healthyfat** [comment]
This is the only one so far that let's me train on 2060RTX 6GB.

https://github.com/d8ahazard/sd_dreambooth_extension/tree/c5cb58328c555ac27679422b1da940a9b19de6f2

**Comment by Zuxier** [comment]
f2eae6127d16a80d1516d3f6245b648eeca26330, 3 days ago,  is good

**Comment by kakis2** [comment]
This combination works for me
Dreambooth revision: "9f4d931a319056c537d24669cb950d146d1537b0"
SD-WebUI revision: "15e89ef0f6f22f823c19592a401b9e4ee477258c"

**Reply by FranciscoFornell** [reply]
Can I ask what graphics card do you have? How much VRAM? For me it's impossible to train anything on the newest commits except lora (with very poor results for training faces unless the model already knows that face, in which case it does improve results). But regular DB training is impossible with a 3080ti 12GB. Everything causes OOM except training SD1.5 with flash attention and fp16 instead of xformers, and even in that case it finishes training but the model learns nothing. I reverted to xformers 0.0.14.dev0 and cu116 and still can't train anything. I'm still testing which was the latest commit that lets me train... If I go back to december it works, but I trained last week with good results too, so there must be a more updated version that works.

**Reply by kakis2** [reply]
I have 24GB graphics card. This revision is different, still working on adjusting settings.

**Comment by yovsac** [comment]
Hello, anybody know how to find the commit number of this video?  for both Stable diffusion and dreambooth? https://www.youtube.com/watch?v=9Nu5tUl2zQw&t=194s  I updated both of them a while back and cant figure out how to go back.
